app/models/package_manager/maven.rb

```diff
@@ -182,8 +177,8 @@ def self.retrieve_versions(versions, name)
             published_at: Time.parse(pom.locate("publishedAt").first.text),
             original_license: license_list,
           }
-        rescue Ox::Error
-          next
+      rescue Ox::Error
+        next
```

Comment:
I think this `rescue` is for the map block itself.

Reply:
I didn't do it, did rubocop break it??

Reply:
Na, just goofy indenting. I don't wanna fight with him 👮‍♂️